packages/example-getting-started/docs/juggler.md

```diff
@@ -1,5 +1,15 @@
 ### Adding the Legacy Juggler
 
+The Legacy Juggler is a "bridge" between the existing
+[loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler)
+and the new LoopBack 4 architecture. It provides the capabilities required to
+access persistence layers/APIs, and perform
+[CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations
+on those sources of data.
+
+It also provides many of the functions and interfaces we'll require for setting
+up our new LoopBack application, which is why we're starting here.
+
 Jump into the directory for your new application. You'll see a folder structure
 similar to this:
 ```
```

Comment:
This folder structure doesn't match the one on the previous step. I think it's best to remove this entirely since the folder structure is shown in the previous step with detailed explanations! (Awesome job with that btw)